front/temporal/labs/activities.ts

```diff
@@ -146,14 +147,21 @@ export async function processTranscriptActivity(
   }
 
   localLogger.info(
```

Comment:
`fileId` is already passed to the `localLogger` at creation. You might want to add the userId in the `localLogger` though.

Reply:
Noice. Changed.